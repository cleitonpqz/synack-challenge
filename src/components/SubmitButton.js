import React from 'react';

const SubmitButton = () => {
  const handleClick = () => {
    console.log('search');
  };

  return (
    <div>
      <button className="button" type="submit" onClick={handleClick}>
        Search
      </button>
    </div>
  );
};

export default SubmitButton;
