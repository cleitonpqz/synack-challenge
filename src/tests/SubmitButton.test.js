import React from 'react';
import { render } from './test-utils';
import SubmitButton from '../components/SubmitButton';

it('should render SubmitButton correctly', () => {
  const component = render(<SubmitButton />);
  expect(component).toMatchSnapshot();
});
