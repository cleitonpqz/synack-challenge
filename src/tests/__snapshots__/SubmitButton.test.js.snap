// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render SubmitButton correctly 1`] = `
<div>
  <button
    className="button"
    onClick={[Function]}
    type="submit"
  >
    Search
  </button>
</div>
`;
